Test-Source/Invoke-DockerPull.Tests.ps1

```diff
@@ -27,115 +19,90 @@ Describe 'Pull docker images' {
 
         It 'pulls public docker image by image name only' {
             Invoke-DockerPull -ImageName 'ubuntu'
-            $result = GetMockValue -Key "pull"
+            $result = GetMockValue -Key "command"
```

Comment:
I suggest we also make the key name a constant (the name is long but at least it's descriptive):

```suggestion
            $result = GetMockValue -Key $Global:InvokeCommandReturnValueKeyName
```

Reply:
Looks like a quick refactoring at global scale. I'll see what i can do this week.

Reply:
Done.